source/NVDAObjects/window/winword.py

```diff
@@ -1112,6 +1112,11 @@ def _normalizeControlField(self, field):
 				field["name"] = name
 				field["alwaysReportName"] = True
 				field["role"] = controlTypes.Role.FRAME
+		collapsedState = field.pop("collapsedState", None) == "true"
+		if collapsedState:
```

Comment:
Why does this need to be assigned to a variable?
```suggestion
		if field.pop("collapsedState", None) == "true":
```